src/System.IO.Compression/tests/CompressionStreamUnitTests.Gzip.cs

```diff
@@ -58,40 +59,136 @@ public void ConcatenatedGzipStreams()
             }
         }
 
+        /// <summary>
+        /// A derived MemoryStream that avoid's MemoryStream's fast path in CopyTo
+        /// that bypasses buffering.
+        /// </summary>
+        class DerivedMemoryStream : MemoryStream
+        {
+            public DerivedMemoryStream() : base()
```

Comment:
Nit: the `: base()` isn't necessary... can just be `public DerivedMemoryStream() { }`